### PR TITLE
[do not merge yet] add logo :)

### PR DESCRIPTION
**Pull request by runcom**
Signed-off-by: Antonio Murdaca <runcom@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 skopeo [![Build Status](https://travis-ci.org/projectatomic/skopeo.svg?branch=master)](https://travis-ci.org/projectatomic/skopeo)
 =
 
+![skopeo logo](https://cdn.rawgit.com/runcom/skopeo-1/add-logo/logo/skopeo.svg)
+
 _Please be aware `skopeo` is still work in progress and it currently supports only registry API V2_
```
Comment (TomSweeneyRedHat) [inline README.md]: I know it's not this change, but please add an 'a' before "work in progress"

Comment (runcom) [reply README.md]: > I know it's not this change, but please add an 'a' before "work in progress"

can you send a PR to fix that instead?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 skopeo [![Build Status](https://travis-ci.org/projectatomic/skopeo.svg?branch=master)](https://travis-ci.org/projectatomic/skopeo)
 =
 
+![skopeo logo](https://cdn.rawgit.com/runcom/skopeo-1/add-logo/logo/skopeo.svg)
```
Comment (mtrmac) [inline README.md]: (Pedantic reminder: this will probably need to point to the project atomic repo in the final version.)

Comment (mtrmac) [reply README.md]: Consider using a raw `` with width/height, or a smaller image; having only the logo above the fold may be a bit much.

File: logo/skopeo.svg [hunk 3]
```diff
@@ -0,0 +1,998 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (mtrmac) [inline logo/skopeo.svg]: (I know, nothing like a clearly OK PR for bikeshedding…)

Consider using the `docs` directory, to have one less top-level item to deal with in directory listings, IDEs and the like.

File: README.md (hunk 2 above)
Comment (runcom) [reply README.md]: :+1: yeah btw, I'll switch to master :)

File: logo/skopeo.svg (hunk 3 above)
Comment (runcom) [reply logo/skopeo.svg]: we keep logo in svg in cri-o `logo/` dir as well

**Comment by rhatdan** [comment]
Shouldn't the logo file be part of this request?  And just reference it locally?

**Comment by runcom** [comment]
> Shouldn't the logo file be part of this request? And just reference it locally?

isn't it? and btw, it's linked to my repo just to show it :)

**Comment by rhatdan** [comment]
@runcom Lets get this updated.

**Comment by runcom** [comment]
> @runcom Lets get this updated.

I need Mairin to provide the fixed logo, will ping her later today hopefully

**Pull request merged**